src/Particular.CodeRules/Particular.ruleset

```diff
@@ -230,4 +230,54 @@
+    <Rule Id="SA1203" Action="None" />             <!-- Constants should appear before fields -->
+    <Rule Id="SA1204" Action="None" />             <!-- Static elements should appear before instance elements -->
+    <Rule Id="SA1205" Action="None" />             <!-- Partial elements should declare access -->
+    <Rule Id="SA1208" Action="None" />             <!-- System using directives should be placed before other using directives -->
+    <Rule Id="SA1214" Action="None" />             <!-- Readonly fields should appear before non-readonly fields -->
+    <Rule Id="SA1302" Action="None" />             <!-- Interface names should begin with I -->
+    <Rule Id="SA1306" Action="None" />             <!-- Field names should begin with lower-case letter -->
+    <Rule Id="SA1307" Action="None" />             <!-- Accessible fields should begin with upper-case letter -->
+    <Rule Id="SA1311" Action="None" />             <!-- Static readonly fields should begin with upper-case letter -->
+    <Rule Id="SA1314" Action="None" />             <!-- Type parameter names should begin with T -->
+    <Rule Id="SA1400" Action="None" />             <!-- Access modifier should be declared -->
+    <Rule Id="SA1401" Action="None" />             <!-- Fields should be private -->
+    <Rule Id="SA1402" Action="None" />             <!-- File may only contain a single type -->
+    <Rule Id="SA1403" Action="None" />             <!-- File may only contain a single namespace -->
+    <Rule Id="SA1413" Action="None" />             <!-- Use trailing comma in multi-line initializers -->
+    <Rule Id="SA1501" Action="None" />             <!-- Statement should not be on a single line -->
+    <Rule Id="SA1502" Action="None" />             <!-- Element should not be on a single line -->
+    <Rule Id="SA1503" Action="None" />             <!-- Braces should not be omitted -->
+    <Rule Id="SA1505" Action="None" />             <!-- Opening braces should not be followed by blank line -->
+    <Rule Id="SA1507" Action="None" />             <!-- Code should not contain multiple blank lines in a row -->
+    <Rule Id="SA1513" Action="None" />             <!-- Closing brace should be followed by blank line -->
+    <Rule Id="SA1515" Action="None" />             <!-- Single-line comment should be preceded by blank line -->
+    <Rule Id="SA1516" Action="None" />             <!-- Elements should be separated by blank line -->
+    <Rule Id="SA1602" Action="None" />             <!-- Enumeration items should be documented -->
+    <Rule Id="SA1611" Action="None" />             <!-- Element parameters should be documented -->
+    <Rule Id="SA1615" Action="None" />             <!-- Element return value should be documented -->
+    <Rule Id="SA1618" Action="None" />             <!-- Generic type parameters should be documented -->
+    <Rule Id="SA1629" Action="None" />             <!-- Documentation text should end with a period -->
```

Comment:
I think for this one we [currently have a test](https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core.Tests/DocumentationTests.cs#L32) that verifies that so why not enable it?

Reply:
We saw _lots_ of violations in core.

Reply:
@danielmarbach we investigated this and it turns out to be a bit complex:

- `DocumentationTests.EnsureNoDocumentationIsEmpty` seems to be quite aggressive with ignoring things that StyleCop does not:
  - It ignores something like `/// See <see cref="Feature.Setup" />`
  - It ignores places where `<code>` is used incorrectly instead of `<c>`.
  - It ignores places where `<code>` is used incorrectly as a top-level element.
  - It ignores documentation on internal types.

We have made a note against that rule, to remind us to remove that test if/when we enable the rule.